src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,61 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Class which counting down, which is working like a stack.
+ */
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int MIN = -1;
+    private static final int MAX = 11;
+    private static final int MAX_SIZE_NUMBERS = 12;
+    private int[] numbers = new int[MAX_SIZE_NUMBERS];
 
-    public int total = -1;
+    private int total = -1;
 
-    public void countIn(int in) {
+    /**
+     * @return actual number elements of stacks;
+     */
+    public int getTotal() {
+        return total;
+    }
+
+    /**
+     * @param IN number which is using to fill the stack.
+     */
+    public void countIn(final int IN) {
         if (!isFull())
-            numbers[++total] = in;
+            numbers[++total] = IN;
     }
 
+    /**
+     * @return return true, if stack is empty and false, if it's not.
+     */
     public boolean callCheck() {
-        return total == -1;
+        return total == MIN;
     }
 
+    /**
+     * @return return true, if stack is full and false, if stack is not.
+     */
     public boolean isFull() {
-        return total == 11;
+        return total == MAX;
     }
 
+    /**
+     * @return return -1, if stack is empty or actual element, if its not.
+     */
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return MIN;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)